apps/examples/vite.config.ts

```diff
@@ -2,6 +2,32 @@ import react from '@vitejs/plugin-react-swc'
 import path from 'path'
 import { PluginOption, defineConfig } from 'vite'
 
+const PR_NUMBER = process.env.VERCEL_GIT_PULL_REQUEST_ID
+
+function getEnv() {
+	if (!process.env.VERCEL_ENV) {
+		return 'local'
+	}
+	if (PR_NUMBER !== undefined && PR_NUMBER !== '') {
+		return 'preview'
+	}
+	if (process.env.VERCEL_ENV === 'production') {
+		return 'production'
+	}
+	return 'canary'
+}
+
+const env = getEnv()
+
+const TLDRAW_BEMO_URL =
+	env === 'production'
+		? 'https://demo.tldraw.xyz'
+		: env === 'canary'
+			? 'https://canary-demo.tldraw.xyz'
+			: env === 'preview'
+				? `https://pr-${PR_NUMBER}-demo.tldraw.xyz`
+				: 'http://localhost:8989'
```

Comment:
can you grab `urlOrLocalFallback` from dotcom's `vite.config.ts`? locally if you're e.g. testing on a mobile device via the local network `localhost` won't work.